src/Resources/skeleton/resetPassword/ResetPasswordController.tpl.php

```diff
@@ -35,8 +39,13 @@ public function __construct(ResetPasswordHelperInterface $resetPasswordHelper)
     /**
      * Display & process form to request a password reset.
      *
+<?php if ($use_attributes) { ?>
+     */
+    #[Route('', name: 'app_forgot_password_request')]
```

Comment:
the empty `''` path was added on purpose when we first created this maker - but I don't remember exactly why we decided to do this. Will investigate and adjust accordingly under a separate issue.

Reply:
I also can't remember... it may have been that `/` here would have made the final URL have a trailing slash - `/reset-password/`